distribution/pom.xml

```diff
@@ -155,6 +155,11 @@
             <artifactId>org.openhab.binding.vitotronic</artifactId>
             <version>${project.version}</version>
         </dependency>
+        <dependency>
```

Comment:
This can be removed now, instead you can add a Karaf feature for it in https://github.com/openhab/openhab2-addons/blob/master/features/openhab-addons/src/main/feature/feature.xml